python/ray/tune/examples/mlflow_example.py

```diff
@@ -85,12 +85,33 @@ def tune_decorated(mlflow_tracking_uri, finish_fast=False):
     parser = argparse.ArgumentParser()
     parser.add_argument(
         "--smoke-test", action="store_true", help="Finish quickly for testing")
+    parser.add_argument(
+        "--tracking-uri",
+        type=str,
+        help="The tracking URI for the MLflow "
+        "tracking server.")
+    parser.add_argument(
+        "--server-address",
+        type=str,
+        default=None,
+        required=False,
+        help="The address of server to connect to if using "
+        "Ray Client.")
     args, _ = parser.parse_known_args()
 
+    if args.server_address:
+        import ray
+        ray.util.connect(args.server_address)
+
+    if args.server_address and not args.tracking_uri:
+        raise RuntimeError("If running this example with Ray Client, "
+                           "the tracking URI for your tracking server should"
+                           "be explicitly passed in")
```

Comment:
nit: can you punctuate these error messages properly (with a period at end of message)?